server.js

```diff
@@ -190,6 +205,7 @@ app.get('/prototype-admin/download-latest', function (req, res) {
 if (useDocumentation) {
   // Copy app locals to documentation app locals
   documentationApp.locals = app.locals
+  documentationApp.locals.serviceName = 'Prototype Kit'
```

Comment:
this needs to be changed to how we've amended it in beta
  documentationApp.locals = Object.assign({}, app.locals)
  documentationApp.locals.serviceName = 'Prototype Kit'

Reply:
dunno if its github being weird, but the line looks how you suggested already?
`documentationApp.locals = Object.assign({}, app.locals)`

Reply:
ah it's because you commented on line 199 instead of 198 :)